examples/max31856_example.cpp

```diff
@@ -51,14 +55,19 @@ ReactESP app([]() {
   // Create a sensor that is the source of our data, that will be read every
   // 1000 ms.
   const uint readDelay = 1000;
```

Comment:
Should be read_delay to be consistent with the style sheet, right?

Reply:
Sharp eyes! Missed that. Thanks.